ui/app/src/components/CreateSiteDialog/CreateSiteDialog.tsx

```diff
@@ -625,7 +626,7 @@ function CreateSiteDialog(props: CreateSiteDialogProps) {
         patch: site.blueprint.version.patch
       }
     };
-    if (site.sandboxBranch) params.sandboxBranch = site.sandboxBranch;
+    if (site.gitBranch) params.sandboxBranch = site.gitBranch as string;
```

Comment:
Why is it required to cast `gitBranch`? Is the type not well defined?
Also, if this is not required, does the API accept blank/missing for sandboxBranch and/or remoteBranch? Don't we need to default it?

Reply:
When the fields are not sent (sandbox and remote) the service use `master` as default.